foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,44 +1,80 @@
+
+#### Step 3: Play round function
+
+Your game will be played round by round. You will make a function that takes the human and computer player choices as arguments, plays a single round and returns a winner announcement:
+
+1. Create a new function named `playRound`.
+1. Define two parameters for `playRound`: `playerChoice` and `computerChoice`. Use these two parameters to take the human and computer choices as arguments.
+1. Make your function's `playerSelection` parameter case-insensitive so that players can input "rock", "ROCK", "RocK", or other variations.
+1. Write the code for your `playRound` function to `return` a string value representing the round winner, such as: "You lose! Paper beats Rock".
+
+Example code:
+
+```javascript
+function playRound(playerChoice, computerChoice) {
+  // your code here!
+}
+
+const playerSelection = "rock";
+const computerSelection = getComputerChoice();
 
-1. Start a new Git repo for your project.
-1. Create a blank HTML document with a script tag (Hint: it is best practice to link an external .js file). This game is going to be played completely from the console, so don't worry about putting anything else in there.
-1. Your game is going to play against the computer, so begin with a function called `getComputerChoice` that will randomly return either 'Rock', 'Paper' or 'Scissors'. We'll use this function in the game to make the computer's play. *Tip: use the console to make sure this is returning the expected output before moving to the next step!*
-1. Write a function that plays a single round of Rock Paper Scissors. The function should take two parameters - the `playerSelection` and `computerSelection` - and then return a string that declares the winner or tie of the round like so: `"You Lose! Paper beats Rock"`
+console.log(playRound(playerSelection, computerSelection));
+```
 
-   - Make your function's playerSelection parameter case-insensitive (so users can input `rock`, `ROCK`, `RocK` or any other variation).
+#### Step 4: Play game function
 
-1. **Important note:** you want to `return` the results of this function call, *not* `console.log()` them. You're going to use what you `return` later on, so let's test this function by using console.log to see the results:
+Your game will play 5 rounds. Write a function named `playGame` that uses `playRound` to play 5 rounds, keeps score, and declares a winner at the end:
 
-   ```javascript
-   function playRound(playerSelection, computerSelection) {
-     // your code here!
-   }
+1. Create a new function named `playGame`.
+1. Get the human player choice using the `prompt` method. Read this [MDN article about the prompt method](https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt) learn more about it.
+1. Put the previous `playRound` function inside `playGame`.
```

Comment:
Is the intent with this line here that users should literally move their playRound declaration inside of playGame, or just that they should call playRound inside of playGame? The wording makes it sound like the former which we probably don't want users to do. We could honestly probably just remove this step since we're already telling users to call playRound 5x on the next line.

Reply:
This depends on where we want the user to declare the scores. If the scores are declared inside the playGame, we won't be able to increment the scores inside the playRound. We can use the return of playRound to conditionally increment the winner's score variable declared in playGame, but this makes the project much more complicated.

So, I thought about those two simple ending outcomes we want to guide the Odinites to: https://gist.github.com/cakegod/2f5f63295ff606a1a729f67564cd7d43.

Reply:
Another option would be declaring the scores as global variables, though yeah as the project is currently written and remains written here the intent seems to be that playGame houses the scores.

If you feel having the playRound moved inside of playGame is the better option I'm good with that. Maybe just tweak the verbiage of this instruction to, "Move your `playRound` function so that it's declared inside of the new `playGame` function"